libandroid-navigation-ui/build.gradle

```diff
@@ -69,6 +69,9 @@ dependencies {
   // AWS Polly
   implementation dependenciesList.polly
 
+  // Picasso
+  implementation dependenciesList.picasso
```

Comment:
If we are using Picasso for one specific use case, it might be better just handling image loading ourselves to eliminate an additional dependency. Otherwise, I do think Picasso is one of the smallest image loading SDKs out there.